api/route/router.ts

```diff
@@ -37,8 +38,11 @@ export default function createRouter(namespace: string = '', operations: Operati
     return router;
   }, Router());
 }
-export function useRouter(instance: ExpressInstance, namespace: string = '/', router: Router) {
-  instance.use('/', router);
+export function useRouter(instance: ExpressInstance, namespace: string = '/', router: Router, authenticated: boolean = false) {
```

Comment:
Since we check for routes with no permissions, we may as well always run the requests though the middleware and remove the 3rd argument.

```ts
instance.use('/', authRoute, router);
```

Reply:
You are right, will fix that